__tests__/app.test.js

```diff
@@ -178,4 +179,47 @@ describe("GET/api/articles/:article_id", () => {
         expect(res.body.message).toBe("Bad Request");
       });
   });
+  test.only("get 201 response with a new posted comment", () => {
+    const newComment = {
+      userName: "lurker",
+      body: "Good work on the article",
+    };
+    return request(app)
+      .post("/api/articles/3/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+
+        console.log(response.body)
+        expect(response.body.message).toBe("Good work on the article");
+      });
+  });
+
+  test("response with the status of 404 and an appropriate error message if invalid username is given", () => {
+    const newComment = {
+      userName: "Tanim",
+      body: "Good work on the article",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(404)
+      .then((res) => {
+        expect(res.body.message).toBe("Not Found");
+      });
+  });
+  test("response with the status of 404 and an appropriate error message if  article id  is not within range", () => {
+    const newComment = {
+      username: "lurker",
+      body: "This is a test comment",
+    };
+
+    return request(app)
+      .post("/api/articles/9999/comments")
+      .send(newComment)
+      .expect(404)
+      .then((res) => {
+        expect(res.body.message).toBe("Not Found");
+      });
```

Comment:
Good error tests! You can also do tests for - 
-  /api/articles/not-a-number/comments for example, that should give a 400.
- as well as the username being wrong, there could be an error where the posted comment is incomplete in some way. maybe a value missing or the wrong data type passed for the body etc.